### ISSUE TITLE
Creating a gist fails

### ISSUE THREAD

**Issue by BigBlueHat** [json-ld/json-ld.org#824]
```json
{
  "message": "Requires authentication",
  "documentation_url": "https://docs.github.com/rest/gists/gists#create-a-gist"
}
```

Not sure how to get around that...

**Comment by gkellogg** [comment]
What’s the context? Is this from the playground?

**Comment by BigBlueHat** [comment]
Yes. Sorry! The gist button on the playground doesn't work and that error is thrown in the browser console.
________________________________

**Comment by davidlehn** [comment]
I haven't used that button in forever.  Did it used to work without authentication?  The docs seem to imply auth is required now.

**Comment by davidlehn** [comment]
Related: https://github.blog/2018-02-18-deprecation-notice-removing-anonymous-gist-creation/
Has it been broken this long?  Not sure if there's a way to open a page with the content to create a non-anon gist.
